SalesforceSDK/Core/Rest/ApiVersionStrings.cs

```diff
@@ -29,7 +29,7 @@ namespace Salesforce.SDK.Rest
 {
     public class ApiVersionStrings
     {
-        public const string VersionNumber = "v31.0";
+        public const string VersionNumber = "v35.0";
```

Comment:
@bhariharan I was thinking for the prefs, we could do something like what we do for bootconfig.json. Does that sound like a good idea?